torch/__init__.py

```diff
@@ -270,22 +270,30 @@ def set_default_tensor_type(t):
 
 
 def set_default_dtype(d):
-    r"""Sets the default floating point dtype to :attr:`d`. This type will be
-    used as default floating point type for type inference in
-    :func:`torch.tensor`.
+    r"""Sets the default floating point dtype to :attr:`d`.
+    This type is also used:
+    1. As default floating point type for type inference in :func:`torch.tensor`.
```

Comment:
this could be a little clearer (I know it's coming from the prior writeup).  But "default floating point type" isn't really defined yet.  Maybe something like:
"As the dtype inferred for python floats" (and similar for complex).

Reply:
updated